acceptance/tests/external_ca_support/jetty_external_root_ca.rb

```diff
@@ -20,18 +20,24 @@
 step "Copy certificates and configuration files to the master..."
 fixture_dir = File.expand_path('../fixtures', __FILE__)
 testdir = master.tmpdir('jetty_external_root_ca')
+backupdir = master.tmpdir('jetty_external_root_ca_backup')
 fixtures = PuppetX::Acceptance::ExternalCertFixtures.new(fixture_dir, testdir)
 
 jetty_confdir = master['puppetserver-confdir']
 
 # Register our cleanup steps early in a teardown so that they will happen even
 # if execution aborts part way.
 teardown do
-  step "Restore /etc/hosts and webserver.conf"
-  on master, "cp -p '#{testdir}/hosts' /etc/hosts"
-  on master, "cp -p '#{testdir}/webserver.conf.orig' '#{jetty_confdir}/webserver.conf'"
+  step "Restore /etc/hosts and puppetserver configs"
+  on master, "cp -p '#{backupdir}/hosts' /etc/hosts"
+  on master, "cp -rp '#{backupdir}/puppetserver/' '#{jetty_confdir}/../'"
```

Comment:
Thanks for the catch. That was a dumb mistake on my part. I have chosen to use \cp to ensure that they system alias that includes the `-i` flag is subverted. This will allow the `--force` to perform the overwrite without interruption.